pkg/oc/admin/diagnostics/diagnostics/cluster/master_node.go

```diff
@@ -146,7 +146,7 @@ func resolveServerIP(serverUrl string, fn dnsResolver) ([]string, error) {
 
 func searchNodesForIP(nodes []kapi.Node, ips []string) types.DiagnosticResult {
 	r := types.NewDiagnosticResult(MasterNodeName)
-	r.Debug("DClu3005", fmt.Sprintf("Seaching for a node with master IP: %s", ips))
+	r.Debug("DClu3005", fmt.Sprintf("Searching for a node with master IP: %s", ips))
```

Comment:
This debug message currently shows:

```
debug: map[origin-node:{origin-node false false false 0} openvswitch:{openvswitch true false true 0} iptables:{iptables true true true 0} docker:{docker true true true 0} etcd:{etcd true true true 0} kubernetes:{kubernetes false false false 0} origin-master:{origin-master false false false 0} atomic-openshift-master:{atomic-openshift-master false false false 0} atomic-openshift-node:{atomic-openshift-node true true true 0}]
```

Actually I wanted to delete it, but if I deleted it, the number `DClu3005` will be absent in master_node.go

Reply:
There's no special reason to keep an identifier.

But, I think you are talking about the debug statement in [`locate_units.go`](https://github.com/openshift/origin/pull/18654/files#diff-3386a4c35931b7de4411edd0a66cc191R41)

Reply:
I don't think you need to keep the message just to preserve `DClu3005` tag. If the tag is present then it should point to unique error/debug but if the error/debug is not relevant any more then we should delete the error/debug. Don't keep stale code just to maintain these tags (we have done the same thing for networkcheck and aggregatedlogging diagnostics).

Reply:
We can always pinpoint the error/debug/warning *without* these tags, we just need ose release (`oc version`) and filepath:line-number which are available. I think these tags are more confusing than useful. May be we should get rid of them?
@sosiouxme

Reply:
The tags certainly serve no purpose for debug/info level at this time. The purpose in warn/error is to have something stable and semantically meaningless that's safe to grep in automation, as there is no machine-readable output at this time. The tags were originally intended for machine-readable output, which is a requested and long-neglected feature.

I doubt anyone's using the tags for that purpose now. I suppose overall it may be better to get rid of tags entirely. It would not be hard to retrofit them when fleshing out the machine-readable feature.

Reply:
I'm sorry, I made a comment under wrong line. As @sosiouxme, I meant `logger.Debug("DS1003", fmt.Sprintf("%v", systemdUnits))`. (And I removed it now.)

The message `DClu3005` is really necessary until this bug was fixed https://bugzilla.redhat.com/show_bug.cgi?id=1368865